README.md

```diff
@@ -0,0 +1,40 @@
+Valkey RFC
+==========
+
+This is a collection of feature proposals and descriptions of changes to Valkey
+that require some more details than just the text in a pull request or an issue.
+It is loosely inspired by RFCs and by Python's enhancement proposals (PEP).
+
+Each feature or larger topic is described in a markdown file that's named in
+uppercase and ends in `.md`. These files are not formally numbered, but we use
+the pull request number that added an RFC to refer to the change. For example,
+this description in the README.md file was written in RFC #1.
+
+Workflow
+--------
+
+An RFC starts off as a pull request. It's reviewed for formatting, style and
+consistency. Then the proposal is merged. This doesn't mean that the feature is
+approved for inclusion in Valkey. It's still just a proposal.
+
+Each file has a status like "draft", "suggested", "approved" or "rejected". The
+possible statuses are yet to be decided.
+
+The core team can later change the status and make changes. For larger changes,
+the PR making the change is mentioned too and can be referred to by their
+respective pull-request numbers.
+
+What's useful to include?
+-------------------------
+
+This is not a strict requirement of what to include in each file. What's useful
+to include may vary from case to case.
```

Comment:
Why not make it strict?

Reply:
We want it to have less friction, be less formal than PEPs and RFCs right?

I think we can require this content but we don't have to enforce a specific order and headings within the document IMO. If somebody already has a document (like RDMA.md, TRIGGERS.md, CLUSTER.md) it can be added without too much rewrite.

I'll rewrite this part.

Reply:
I guess I would just like to have a strict set of stuff you "must" have, but give a lot of flexibility in those sections.

Reply:
Please suggest a formulation.